### DISCUSSION TITLE
python version support

### DISCUSSION THREAD

**Discussion by darrahts** [nasa/prog_models#402] [General]
@teubert 

Are there known compatibility issues with python 3.6? In my use I haven't ran into any issues, but have not thoroughly tested. 

https://github.com/nasa/prog_models/blob/44c4524335dd1e1042a32594183e362dbf9d35b0/setup.py#L40

**Comment by teubert** [comment]
Python3.6 is depreciated, so according to our security policy, we can no longer support it. 

That said, there are no known issues, but we no longer test with 3.6, so I can't say for sure

**Comment by darrahts** [comment]
Got it. I also just realized I was using an old environment. Thanks!
